Gems/DiffuseProbeGrid/Code/Source/EditorComponents/EditorDiffuseProbeGridComponent.cpp

```diff
@@ -34,7 +34,7 @@ namespace AZ
             if (AZ::SerializeContext* serializeContext = azrtti_cast<AZ::SerializeContext*>(context))
             {
                 serializeContext->Class<EditorDiffuseProbeGridComponent, BaseClass>()
-                    ->Version(2, ConvertToEditorRenderComponentAdapter<1>)
+                    ->Version(3, ConvertToEditorRenderComponentAdapter<1>) // added emissive multiplier
```

Comment:
Why does the version need to update? I don't see any changes in the serialized fields

Reply:
I didn't update the version when I added the emissive multiplier, so fixing that now.